autogpt_platform/backend/backend/blocks/compass/triggers.py

```diff
@@ -0,0 +1,59 @@
+from pydantic import BaseModel
+
+from backend.data.block import (
+    Block,
+    BlockCategory,
+    BlockManualWebhookConfig,
+    BlockOutput,
+    BlockSchema,
+)
+from backend.data.model import SchemaField
+from backend.integrations.webhooks.compass import CompassWebhookType
+
+
+class Transcription(BaseModel):
+    text: str
+    speaker: str
+    end: float
+    start: float
+    duration: float
+
+
+class TranscriptionDataModel(BaseModel):
+    date: str
+    transcription: str
+    transcriptions: list[Transcription]
+
+
+class CompassAITriggerBlock(Block):
+    class Input(BlockSchema):
+        payload: TranscriptionDataModel = SchemaField(hidden=True)
+
+    class Output(BlockSchema):
+        transcription: str = SchemaField(
+            description="The transcription of the compass transcription."
+        )
+
+    def __init__(self):
+        super().__init__(
+            id="9464a020-ed1d-49e1-990f-7f2ac924a2b7",
+            description="This block forwards an input value as output, allowing reuse without change.",
+            categories={BlockCategory.HARDWARE},
```

Comment:
is it?

Reply:
cc @Bentlybro

Reply:
Id say ``HARDWARE`` is right for this, it is technically an AI assistant but its also mainly a hardware based device that the system would be interacting with, its a mix of both but i feel ``HARDWARE`` is a good choice